cli/core/hack/choco/build_package.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "$(command -v choco)" ]; then
```

Comment:
This line means we require not only choco to be installed but also the script is run in a *nix type os or shell (e.g. git-bash on windows)

Reply:
Good question.  I guess it will depend on our final solution for automating the build of the Chocolatey package.  If we do it in a Github action we can use an Ubuntu machine.  We could of course change the script to run on Windows, but I liked that using docker we could run it on our workstations more easily.